src/sass/app_2/_common/components/account-switcher.scss

```diff
@@ -101,40 +113,77 @@
         &-title {
             @include typeface(--small-center-normal-grey, none);
             @include toEm(padding, 8px 16px, 1.2em);
-            border: 1px dashed $COLOR_LIGHT_BLACK_2;
             width: 100%;
             border-radius: 4px;
             cursor: pointer;
+            @include themify($themes) {
```

Comment:
@cakasuma just curious, do we have to include `themify` for individual classes? since`themify` was already added to the top-most parent (the `.body`), shouldn't using `themed('color_var')` suffice to change the colors (without having to include `themify` again)?

Reply:
@kellybinary themify works as a mixin to produce the current selector with the object themes provided. basically themify function will add the `theme--dark` class plus the current selector. and then themed('color_var'). and yes, we can put the @include themify to the most parent for example here `.acc-switcher` and just use themed('color_var') all the way to its child. however. if we do that, the file of css will be very high since it will doubled the styling.

Reply:
you mean, it's better to use `@include themify` for all parent?

Reply:
i mean, it can be used for all parent. however it will doubled the css yes. that is why i believe its better to put it under individual class.